db/migrate/20230109080950_rename_solidus_paypal_braintree_source_type.rb

```diff
@@ -0,0 +1,16 @@
+class RenameSolidusPaypalBraintreeSourceType < ActiveRecord::Migration[6.1]
```

Comment:
Should we also add a migration for changing the table names? Or are we aiming at handling everything with the table_name_prefix configuration?

Reply:
As I mentioned in https://github.com/solidusio/solidus_braintree/pull/99#discussion_r1070197261, I'm inclined to remove `table_name_prefix` in favor of new migrations renaming the tables.

Reply:
I think the migration should be enough, as long as it's crafted carefully, with stores that can have a huge number of records in mind (the `strong_migrations` gem can help in doing that).

Reply:
@kennyadsl @elia Hmm... I haven't considered the potential effect of a table name to a large production database. `strong_migrations` says that renaming a table is potentially dangerous, so I'd rather leave it to the extension user to migrate their own database, rather than providing them a migration that can negatively impact them. 

I'll no longer continue with the table rename migration and will just keep the `table_name_prefix`.